syntax_and_semantics/annotations/user_defined_annotations.md

```diff
@@ -0,0 +1,173 @@
+# User Defined Annotations
+
+Users can define their own annotations using the `annotation` keyword, which works similarly to defining `class` or `sruct`.
+
+```crystal
+annotation MyAnnotation; end
+```
+
+The annotation can then be applied to various types, including:
```

Comment:
The list below does not really include any type.

Reply:
```suggestion
The annotation can then be applied to various items, including:
```
?